src/pocketmine/item/Bucket.php

```diff
@@ -44,6 +46,12 @@ public function onClickBlock(Player $player, Block $block, Block $blockClicked,
 				$player->getServer()->getPluginManager()->callEvent($ev = new PlayerBucketFillEvent($player, $block, $face, $this, $result));
 				if(!$ev->isCancelled()){
 					$player->getLevel()->setBlock($blockClicked, Block::get(Block::AIR), true, true);
+					if($blockClicked instanceof FlowingLava){
+						$soundId = LevelSoundEventPacket::SOUND_BUCKET_FILL_LAVA;
+					}else{
+						$soundId = LevelSoundEventPacket::SOUND_BUCKET_FILL_WATER;
+					}
```

Comment:
@jasonwynn10 Why did you react with confused? You did it directly in your weather PR.